compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/Passes.cpp

```diff
@@ -588,6 +591,8 @@ void addAMDAIEObjectFifoLoweringPasses(OpPassManager &passManager,
   passManager.addPass(createCanonicalizerPass());
 
   passManager.addPass(createAMDAIEDmaCompositionPass());
+  passManager.addPass(createAMDAIECanonicalizeDoublyStridedOpPass());
+  passManager.addPass(createAMDAIEDmaLoopSubsumptionPass());
```

Comment:
Can't we put the canonicalization inside the `DmaComposition` pass to avoid `DmaComposition` -> `Canonicalization` -> `DmaLoopSubsumption`?

Reply:
Done.